### PR TITLE
Fix inconsistent printing of meditation

### PR DESCRIPTION
**Pull request by Iftakharpy**
This fixes the issues I observed while solving the koans. Some of the issues I noticed are:
- Random vertical position of printing koans
- Sometimes koan intro printed before clear screen
- Next koan does not appear when all koan passes for a module

### CODE REVIEW

File: lib/display/progress_bar.ex [hunk 1]
```diff
@@ -4,14 +4,19 @@ defmodule Display.ProgressBar do
...
-    "|" <> String.pad_trailing(arrow, @progress_bar_length) <> "| #{current} of #{total}"
+    "|" <> String.pad_trailing(arrow, @progress_bar_length) <> "| #{current} of #{total} -> #{progress_percentage}% complete"
   end
 
   defp calculate_progress(current, total) do
     round(current / total * @progress_bar_length)
   end
 
+  defp calculate_percentage(current, total) do
+    Float.round(current / total * 100, 1)
```
Comment (felipesere) [inline lib/display/progress_bar.ex:17]: I wonder if this would look better without any decimal places?
```
0 of 12 -> 0% complete
```

Comment (Iftakharpy) [reply lib/display/progress_bar.ex:17]: I thought it  would be useful for people like me who wants to know the progress a little more precisely. But either way it's fine doesn't matter too much.

Comment (iamvery) [reply lib/display/progress_bar.ex:17]: I don't have a strong opinion. I tend to agree that decimals aren't terribly useful in this context.

**Review by iamvery — COMMENTED** [review]
Changes lgtm. Since Felipe is around (and had a hand in writing this afaicr), I'll defer to him for approval.

**Comment by felipesere** [comment]
Thank you for this!

**Pull request merged**